sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1068,6 +1140,16 @@ public WriteResult expand(PCollection<T> input) {
           "No more than one of jsonSchema, schemaFromView, or dynamicDestinations may "
               + "be set");
 
+      if (getTriggeringFrequency() != null) {
+        checkArgument(getMethod() == Method.FILE_LOADS);
```

Comment:
While true for all runners, I don't think triggering is fundamentally incompatible with bounded.